src/wp-includes/functions.php

```diff
@@ -6998,6 +6998,12 @@ function wp_debug_backtrace_summary( $ignore_class = null, $skip_frames = 0, $pr
  * @return int[] Array of IDs not present in the cache.
  */
 function _get_non_cached_ids( $object_ids, $cache_key ) {
+	$object_ids     = array_filter(
+		$object_ids,
+		static function ( $object_id ) {
+			return filter_var( $object_id, FILTER_VALIDATE_INT );
```

Comment:
WordPress doesn't use `filter_var` as it's possible for hosts to disable the Filter extension.

I gather you want to allow `"16"` but not `"16.2"`. I guess you could do something like `(string) (int) $object_id === (string) $object_id` and leave a comment explaining why and begging for forgiveness.

Reply:
Done.

Reply:
The type-cast check also requires an `is_string()` call, otherwise `5.0` would pass 🙃